mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -15,6 +15,10 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	FIELD field_27857 inPowderSnow Z
 	FIELD field_27858 FROZEN_TICKS Lnet/minecraft/class_2940;
 	FIELD field_28628 wasInPowderSnow Z
+	FIELD field_28629 onFire Z
```

Comment:
#2266 calls it `wasOnFire`, which PR is right?

Reply:
This is a field used exclusively in `Entity.move(MovementType, Vec3d)` that tracks if the entity is on fire in the beginning of the movement checks. So #2266 is more correct, but this honestly is only to prevent situation like making tons of extinguishing sound maybe like when you stand on a burning netherrack in rain?